src/engine/specs/CoreSpecs.ts

```diff
@@ -90,6 +91,16 @@ export interface IWallSpec extends IBaseSimObjectSpec {
   height?: number;
 }
 
+/**
+ * Spec for a Perimeter
+ */
+export interface IPerimeterSpec extends IBaseSimObjectSpec {
+  type: "perimeter";
+  walls?: IWallSpec[];
```

Comment:
We can probably do away with this field. Since we're defining a perimeter, we just need to specify the thickness and height

Reply:
I just wanted to retain the ability to define your own walls if you preferred, but I guess that use case is not very common

Reply:
Got it. Yeah I don't think i'll be a super common use case. Plus we have other ways to add walls besides the perimeter call. 

I think we should just have a `perimeter` defined as "we're gonna add walls all around"